monitoring/start.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+node /config-generator/src/generate.js
```

Comment:
For yargs, we can pass in incoming CLI requests with... I think `node /config-generator/src/generate.js "$@"`. But maybe `$*` or `"$*"` or `$@` would work better, I can never keep track of those four